src/terminal/color.lua

```diff
@@ -0,0 +1,301 @@
+--- Module for setting text colors and attributes
```

Comment:
```suggestion
--- Module for setting text colors and attributes.
```
The title needs a '.' to end